painttiming.bs

```diff
@@ -62,9 +86,24 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn export>Contentful</dfn>: An element counts as contentful when one or more of the following apply:
+* The element contains a [=text node=], with at least one character, excluding [=white-space=].
```

Comment:
You may need to link contains here (I assume it means it has a descendant in the DOM tree that is a text node).

Reply:
yes, exactly. Will do that.